### DISCUSSION TITLE
Capture HVAC on / off state to automate duct fans

### DISCUSSION THREAD

**Discussion by CanicheRoyal** [jeffschubert/homebridge-daikin-oneplus#48] [Ideas]
I have a poorly balanced HVAC system that requires duct fans to move air to appropriate parts of the home.  Would be nice if I could have Homekit / Homebridge automatically turn these fans on when the thermostat calls for heat or cooling using a smart outlet.

Homekit automation seems to rely on temperature limits, but those may not be sensitive enough to reliably start and stop the fans.  Is there a way to capture the on/off state for the purpose of homekit automation?  How might that work?

Thanks,

**Comment by jeffschubert** [comment]
A resolution to this is planned in the next release. It will be possible to enable state switches for the possible thermostat states so that automations can trigger off of when the switch changes.
